msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,348 @@
+Reviewing these guidelines can help you avoid issues that frequently cause apps to fail Microsoft Teams store and AppSource submission. These guidelines complement the Microsoft [commercial marketplace certification policies](https://docs.microsoft.com/legal/marketplace/certification-policies) and will be updated frequently to reflect new Teams capabilities, user feedback, and changes to business rules.
 
->[!NOTE]
->**[Section 1140](/legal/marketplace/certification-policies#1140-teams)** is specific to Microsoft Teams and **[sub-section 1140.4](https://docs.microsoft.com/legal/marketplace/certification-policies#11404-functionality)** addresses functionality requirements for Teams apps.
+> [!TIP]
+> Some guidelines may not be applicable to your app. For example, if your app doesn't include a bot, you can ignore bot-related guidelines.
+
+## 1.0 Value proposition
+
+### 1.1 App name
+
+An app's name plays a critical role in how users discover it in the store. Remember the following about app names:
+
+* The name must include terms relevant to your users.
+* It must not copy the name of an existing app in the store or other offer in the commercial marketplace.
+* Common nouns must be prefixed or suffixed with the publisher’s name (for example, “XYZ Tasks” rather than “Tasks”)
+* It must not use **Teams** or other Microsoft product names that could falsely indicate co-branding or co-selling. (For more information about referencing Microsoft software, products, and services, see the [Microsoft Trademark and Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general)).
+* If your app is part of an official partnership with Microsoft, the name of your app must come first (for example, Salesforce Connector for Microsoft Teams).
+
+### 1.2 Suitable for workplace consumption
+
+App content must be suitable for general workplace consumption and abide by all restrictions listed in the [commercial marketplace certification policies](https://docs.microsoft.com/legal/marketplace/certification-policies#10010-inappropriate-content). Content related to religion, politics, gambling, and prolonged entertainment is prohibited.
+
+Your app must facilitate group collaboration, improve an individual's productivity, or both. Apps intended for team bonding and socializing must be collaborative and designed for multiple participants. These types of apps also should not require a substantial time investment or perceptively impact productivity.
+
+### 1.3 Similar platforms and services
+
+Apps must focus on the Teams experience and not include the names, icons, or imagery of other similar chat-based collaboration platforms or services unless the apps provide specific interoperability.
+
+## 2.0 Security
+
```

Comment:
@heath-hamilton, add 2.1 as 

**Publisher Verification & App permissions:**
•	Publisher verification helps admins and end users understand the authenticity of application. Teams app must complete publisher verification for application. A blue "verified" badge appears on the Azure AD consent prompt and other screens. Please refer Mark an app as publisher verified  along with Troubleshoot publisher verification,  FAQ.
•	Apps that request permissions during the Azure Active Directory (Azure AD) consent process must provide business justification for these requests.

Reply:
@ramendhe I added this content but changed the structure slightly to keep things self contained. Let me know what you think.

Reply:
@heath-hamilton, change in structure looks good. But In 2.2.1 Publisher Verification - Please don't omit -

- FAQ link - https://docs.microsoft.com/en-us/azure/active-directory/develop/publisher-verification-overview#frequently-asked-questions
-  A blue "verified" badge requirement on the Azure AD consent prompt and other screens.

Reply:
@heath-hamilton, Also should we avoid 3rd level marking as 2.2.1 & instead use simple bullets for these 2 topics

Reply:
@heath-hamilton, Remove - 2.1 App permissions: 
Apps that request permissions during the Azure Active Directory (Azure AD) consent process must provide business justification for these requests.